src/figuresClasses.ts

```diff
@@ -1,11 +1,89 @@
-export interface Figure {}
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please update the enum values to start with a capital letter to follow the convention.

GOOD EXAMPLE:
enum Color {
  Red = 'red',
  Green = 'green',
  Blue = 'blue',
}